From 6a391474a8398e5fa8ba0d642a43097232a1ab29 Mon Sep 17 00:00:00 2001
From: Godswill Onwusilike <onwusilikegodswill@gmail.com>
Date: Sat, 16 May 2026 21:30:15 +0100
Subject: [PATCH 00065/15333] docs: gpu: drm-uapi: fix spelling of
 "unprivileged"

Correct the spelling of "unpriviledged" to "unprivileged" in DRM uAPI documentation.

Signed-off-by: Godswill Onwusilike <onwusilikegodswill@gmail.com>
Signed-off-by: Thomas Zimmermann <tzimmermann@suse.de>
Link: https://patch.msgid.link/20260516203015.33466-1-onwusilikegodswill@gmail.com
---
 Documentation/gpu/drm-uapi.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/gpu/drm-uapi.rst b/Documentation/gpu/drm-uapi.rst
index 2c2f939322fb..93df92c4ac8c 100644
--- a/Documentation/gpu/drm-uapi.rst
+++ b/Documentation/gpu/drm-uapi.rst
@@ -574,7 +574,7 @@ ENOSPC:
 EPERM/EACCES:
         Returned for an operation that is valid, but needs more privileges.
         E.g. root-only or much more common, DRM master-only operations return
-        this when called by unpriviledged clients. There's no clear
+        this when called by unprivileged clients. There's no clear
         difference between EACCES and EPERM.
 
 ENODEV:
-- 
2.55.0

